aggsender/aggsender.go

```diff
@@ -277,27 +281,33 @@ func (a *AggSender) sendCertificates(ctx context.Context, returnAfterNIterations
 					a.log.Infof("An InError cert exists but skipping send cert because RetryCertAfterInError is false")
 				}
 			}
+
 			if returnAfterNIterations > 0 && iteration >= returnAfterNIterations {
 				a.log.Warnf("reached number of iterations, so we are going to return")
 				return
 			}
 		case epoch := <-chEpoch:
 			iteration++
 			a.log.Infof("Epoch received: %s", epoch.String())
-			checkResult := a.certStatusChecker.CheckPendingCertificatesStatus(ctx)
-			if !checkResult.ExistPendingCerts {
-				_, err := a.sendCertificateWithRetries(ctx)
-				if err != nil {
-					a.log.Errorf("error sending certificate: %v", err)
-					a.status.SetLastError(err)
-				}
+			checkResult, err := a.certStatusChecker.CheckPeriodicallyCertificateStatus(ctx)
+			if err != nil {
+				a.log.Errorf("Epoch trigger: error checking certificate status: %v", err)
+				a.status.SetLastError(err)
 			} else {
-				a.log.Infof("Skipping epoch %s because there are pending certificates",
-					epoch.String())
+				if !checkResult.ExistPendingCerts {
+					_, err := a.sendCertificateWithRetries(ctx)
+					if err != nil {
+						a.log.Errorf("Epoch trigger: error sending certificate: %v", err)
+						a.status.SetLastError(err)
+					}
+				} else {
+					a.log.Infof("epoch trigger: Skipping epoch %s because there are pending certificates",
```

Comment:
Nitpick, make the letters case consistent in the logs (either upper or lower case everywhere):
- https://github.com/agglayer/aggkit/blob/36e896bc31e62d1e3f307303ea897475b3fcc392/aggsender/aggsender.go#L294
- https://github.com/agglayer/aggkit/blob/36e896bc31e62d1e3f307303ea897475b3fcc392/aggsender/aggsender.go#L300

```suggestion
					a.log.Infof("Epoch trigger: Skipping epoch %s because there are pending certificates",
```